lib/browser/guest-view-manager.js

```diff
@@ -73,37 +73,32 @@ var createGuest = function (embedder, params) {
   }
 
   // Destroy guest when the embedder is gone or navigated.
-  destroyEvents = ['will-destroy', 'crashed', 'did-navigate']
-  destroy = function () {
+  const destroyEvents = ['will-destroy', 'crashed', 'did-navigate']
+  const destroy = function () {
     if (guestInstances[id] != null) {
-      return destroyGuest(embedder, id)
+      destroyGuest(embedder, id)
     }
   }
-  for (i = 0, len = destroyEvents.length; i < len; i++) {
-    event = destroyEvents[i]
+  for (const event of destroyEvents) {
     embedder.once(event, destroy)
 
-    // Users might also listen to the crashed event, so We must ensure the guest
+    // Users might also listen to the crashed event, so we must ensure the guest
     // is destroyed before users' listener gets called. It is done by moving our
     // listener to the first one in queue.
-    listeners = embedder._events[event]
+    const listeners = embedder._events[event]
     if (Array.isArray(listeners)) {
       moveLastToFirst(listeners)
     }
   }
   guest.once('destroyed', function () {
-    var j, len1, results
-    results = []
```

Comment:
This was the unused `results` array that was being populated and then thrown away when `destroyed` was fired